canonical/lxd-ui
# Use segmented control and icons for instance detail header actions

## Done

- Use segmented control and icons for instance detail header actions

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Check instance detail page header actions on various screen sizes
